opendcs-rest-api/src/main/java/org/opendcs/odcsapi/res/RestServices.java

```diff
@@ -36,9 +35,10 @@ public RestServices()
 		LOGGER.debug("Initializing odcsapi RestServices.");
 		packages("org.opendcs.odcsapi");
 		String officeId = servletContext.getInitParameter("opendcs.rest.api.cwms.office");
+		String databaseType = servletContext.getInitParameter("editDatabaseType");
 		if(officeId != null)
 		{
-			DbInterface.decodesProperties.setProperty("CwmsOfficeId", officeId);
+			DbInterface.decodesProperties.setProperty("editDatabaseType", databaseType);
```

Comment:
It dawns on me that we could create a new table in the schema that contains the database type and some info, and when the connection is opened that info could be used.

Not sure if that's a good idea, just don't like having to have users edit the context parameters, but I guess that is a normal thing to do.

Reply:
@wjonassen had the same thought. I don't like the context parameters either or the fact that the DbInterface is reading from the decodes.properties/user.properties. I'll make an issue to track. The office id handling will definitely need some more design once the REST API is updated to support CWMS.

Reply:
https://github.com/opendcs/rest_api/issues/191